core/shared/src/test/scala/com/monovore/decline/HelpSpec.scala

```diff
@@ -43,7 +43,9 @@ class HelpSpec extends AnyWordSpec with Matchers {
       }
 
       Help.fromCommand(parser).toString should equal(
-        """Usage: program [--first] [--second <integer>] [--third <integer>] [--flagOpt[=<string>]] --flag[=<string>] [--flag[=<string>]]... run
+        """
+          |Usage:
+          |    program [--first] [--second <integer>] [--third <integer>] [--flagOpt[=<string>]] --flag[=<string>] [--flag[=<string>]]... run
```

Comment:
Got to change this back! (Among other things, the single-line usage statement is extremely standard in the wild.)

Reply:
Reverted begrudgingly to the original format :) https://github.com/bkirwi/decline/pull/539/commits/d8658fbd2c62907023957785f714567ec6ec6eef